specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,310 @@
+            }
+        },
+        "InformationProtectionPolicy": {
+            "type": "object",
+            "description": "Information protection policy",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Information protection policy data",
+                    "$ref": "#/definitions/InformationProtectionPolicyProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                }
+            ]
+        },
+        "InformationProtectionPolicyProperties": {
+            "type": "object",
+            "description": "describes properties of an information protection policy",
+            "properties": {
+                "lastModifiedUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "Describes the last UTC time the policy was modified"
+                },
+                "sensitivityLabels": {
+                    "type": "array",
+                    "description": "List of sensitivity labels",
+                    "items": {
+                        "$ref": "#/definitions/SensitivityLabel"
+                    }
+                },
+                "sensitivityInformationTypes": {
+                    "type": "array",
+                    "description": "List of sensitivity information types",
+                    "items": {
+                        "$ref": "#/definitions/SensitivityInformationType"
+                    }
+                }
+            }
+        },
+        "SensitivityLabel": {
+            "type": "object",
+            "description": "The sensitivity label.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The id of the sensitivty label."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the sensitivity label."
+                },
+                "order": {
+                    "type": "number",
+                    "description": "The order of the sensitivity label."
+                }
+            }
+        },
+        "SensitivityInformationType": {
+            "type": "object",
+            "description": "The information type.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The id of the information type."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the information type."
+                },
+                "order": {
+                    "type": "number",
+                    "description": "The order of the information type."
+                },
+                "recommendedLabelId": {
```

Comment:
Does `recommendedLabelId` have to be a GUID, or can it simply be a string? If it must be a GUID, then it should not be of type string, but typed as a uuid.

Reply:
No, it represents a label ID, and it can be a string.

Reply:
Actually we decided that they should be guids. Will update the type

Reply:
Why does it need to be a GUID? Are you using it for some kind of uniqueness? GUIDs hamper the usability considerably.

Reply:
We're planning to integrate with Microsoft Information Protection and Azure Information Protection in the future, and they work with guids. Also built-in information types have pre-defined guids and they are globally unique.